recipes/pivy/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "pivy" %}
+{% set version = "0.6.4b1" %}
+{% set sha256 = "98743d0daa3b7aaaf33bb055035cf5141679a5ec6f237c4a8ee3c40d594f3510" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}.tar.gz
+    url: https://github.com/FreeCAD/pivy/archive/{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+
+build:
+    number: 0
+    skip: true  # [(win and not py36)]
+
+    features:
+        - vc14  # [(win and py36)]
+
+requirements:
+    build:
+        - toolchain
+        - nomkl  # [unix]
```

Comment:
So I ran across this recently, but couldn't quite understand why `nomkl` was here. Any info?